public/catch-js-errors.js

```diff
@@ -1,5 +1,6 @@
 ( function() {
 	var savedWindowOnError = window.onerror,
+		assignment = 1,
```

Comment:
Seems unnecessary. IMHO it's ok to mix es6 and es5, so I'd just declare this as a constant at the point it's given a value below. Not a blocker.